### PR TITLE
Rename folder keras to legacy-keras

### PR DESCRIPTION
**Pull request by bojiang**

### CODE REVIEW

File: legacy-keras/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Found examples with new keras API & tensorflow 2.0 in /tensorflow
```
Comment (bojiang) [inline legacy-keras/README.md:1]: The highest version of Keras that supports TensorFlow 1.x is 2.3.

**Comment by parano** [comment]
@bojiang you will also need to update all the notebook links in BentoML's readme and docs site, could you create a PR for that so we can merge both sides together and avoid broken links?

File: legacy-keras/README.md (hunk 1 above)
Comment (parano) [reply legacy-keras/README.md:1]: I think KerasModelArtifact actually works tf.keras in TensorFlow 2.0, could you help verify that?

Comment (parano) [reply legacy-keras/README.md:1]: It should support both tf.keras and the standalone keras module.

Comment (bojiang) [reply legacy-keras/README.md:1]: Tensorflow Version: 2.1.0
Keras Version: 2.3.1
BentoML Version: 0.8.3+49.gdcc2e8b

keras-fashion-mnist.ipynb :
```
>>> mnist_svc.predict([test_images[0].squeeze(-1)])
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-17-1ad9b1a21329> in <module>
----> 1 mnist_svc.predict([test_images[0].squeeze(-1)])

~/lab/gallery/legacy-keras/fashion-mnist/keras_fashion_mnist.py in predict(self, imgs)
     21             inputs.append(img)
     22         inputs = np.stack(inputs)
---> 23         class_idxs = self.artifacts.classifier.predict_classes(inputs)
     24         return [class_names[class_idx] for class_idx in class_idxs]

~/BentoML/bentoml/artifact/keras_model_artifact.py in predict_classes(self, *args, **kwargs)
    272         with self.graph.as_default():
    273             with self.sess.as_default():
--> 274                 return self.keras_model.predict_classes(*args, **kwargs)
    275 
    276     def __call__(self, *args, **kwargs):

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/keras/engine/sequential.py in predict_classes(self, x, batch_size, verbose)
    266             A numpy array of class predictions.
    267         """
--> 268         proba = self.predict(x, batch_size=batch_size, verbose=verbose)
    269         if proba.shape[-1] > 1:
    270             return proba.argmax(axis=-1)

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/keras/engine/training.py in predict(self, x, batch_size, verbose, steps, callbacks, max_queue_size, workers, use_multiprocessing)
   1460                                             verbose=verbose,
   1461                                             steps=steps,
-> 1462                                             callbacks=callbacks)
   1463 
   1464     def train_on_batch(self, x, y,

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/keras/engine/training_arrays.py in predict_loop(model, f, ins, batch_size, verbose, steps, callbacks)
    322             batch_logs = {'batch': batch_index, 'size': len(batch_ids)}
    323             callbacks._call_batch_hook('predict', 'begin', batch_index, batch_logs)
--> 324             batch_outs = f(ins_batch)
    325             batch_outs = to_list(batch_outs)
    326             if batch_index == 0:

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/tensorflow_core/python/keras/backend.py in __call__(self, inputs)
   3725         value = math_ops.cast(value, tensor.dtype)
   3726       converted_inputs.append(value)
-> 3727     outputs = self._graph_fn(*converted_inputs)
   3728 
   3729     # EagerTensor.numpy() will often make a copy to ensure memory safety.

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/tensorflow_core/python/eager/function.py in __call__(self, *args, **kwargs)
   1549       TypeError: For invalid positional/keyword argument combinations.
   1550     """
-> 1551     return self._call_impl(args, kwargs)
   1552 
   1553   def _call_impl(self, args, kwargs, cancellation_manager=None):

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/tensorflow_core/python/eager/function.py in _call_impl(self, args, kwargs, cancellation_manager)
   1589       raise TypeError("Keyword arguments {} unknown. Expected {}.".format(
   1590           list(kwargs.keys()), list(self._arg_keywords)))
-> 1591     return self._call_flat(args, self.captured_inputs, cancellation_manager)
   1592 
   1593   def _filtered_call(self, args, kwargs):

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/tensorflow_core/python/eager/function.py in _call_flat(self, args, captured_inputs, cancellation_manager)
   1713               {"PartitionedCall": "PartitionedCallUnused",
   1714                "StatefulPartitionedCall": "PartitionedCallUnused"}):
-> 1715             flat_outputs = forward_function.call(ctx, args_with_tangents)
   1716     forward_backward.record(flat_outputs)
   1717     return self._build_call_outputs(flat_outputs)

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/tensorflow_core/python/eager/function.py in call(self, ctx, args, cancellation_manager)
    572                 executing_eagerly=executing_eagerly,
    573                 config=config,
--> 574                 executor_type=executor_type)
    575 
    576     if executing_eagerly:

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/tensorflow_core/python/ops/functional_ops.py in partitioned_call(args, f, tout, executing_eagerly, config, executor_type)
    857           f=f,
    858           config_proto=config,
--> 859           executor_type=executor_type)
    860     else:
    861       outputs = gen_functional_ops.partitioned_call(

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/tensorflow_core/python/ops/gen_functional_ops.py in stateful_partitioned_call(args, Tout, f, config, config_proto, executor_type, name)
    599         "StatefulPartitionedCall", args=args, Tout=Tout, f=f, config=config,
    600                                    config_proto=config_proto,
--> 601                                    executor_type=executor_type, name=name)
    602   _result = _outputs[:]
    603   if not _result:

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/tensorflow_core/python/framework/op_def_library.py in _apply_op_helper(op_type_name, name, **keywords)
    410               dtype=dtype if dtype else None,
    411               preferred_dtype=default_dtype,
--> 412               as_ref=input_arg.is_ref)
    413           if input_arg.number_attr and len(
    414               set(v.dtype.base_dtype for v in values)) > 1:

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/tensorflow_core/python/framework/ops.py in internal_convert_n_to_tensor(values, dtype, name, as_ref, preferred_dtype, ctx)
   1380             as_ref=as_ref,
   1381             preferred_dtype=preferred_dtype,
-> 1382             ctx=ctx))
   1383   return ret
   1384 

/opt/conda/envs/bentoml-dev-py36/lib/python3.6/site-packages/tensorflow_core/python/framework/ops.py in convert_to_tensor(value, dtype, name, as_ref, preferred_dtype, dtype_hint, ctx, accepted_result_types)
   1278       graph = get_default_graph()
   1279       if not graph.building_function:
-> 1280         raise RuntimeError("Attempting to capture an EagerTensor without "
   1281                            "building a function.")
   1282       return graph.capture(value, name=name)

RuntimeError: Attempting to capture an EagerTensor without building a function.
```

Comment (bojiang) [reply legacy-keras/README.md:1]: While:
Tensorflow Version: 2.2.0
Keras Version: 2.4.3
BentoML Version: 0.8.3

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-16-99e234e372e5> in <module>
      4 # 2) `pack` it with required artifacts
      5 mnist_svc = KerasFashionMnistService()
----> 6 mnist_svc.pack('classifier', model)
      7 
      8 # 3) save your BentoSerivce to file archive

~/BentoML/bentoml/service.py in pack(self, name, *args, **kwargs)
    805         :return: this BentoService instance
    806         """
--> 807         self.artifacts.get(name).pack(*args, **kwargs)
    808         return self
    809 

~/BentoML/bentoml/artifact/artifact.py in wrapped_pack(*args, **kwargs)
    120                         "behaviors"
    121                     )
--> 122                 ret = original(*args, **kwargs)
    123                 # do not set `self._pack` if `pack` has failed with an exception raised
    124                 self._packed = True

~/BentoML/bentoml/artifact/keras_model_artifact.py in pack(self, data)
    188 
    189         self.bind_keras_backend_session()
--> 190         model._make_predict_function()
    191 
    192         self._model = model

AttributeError: 'Sequential' object has no attribute '_make_predict_function'
```

**Comment by bojiang** [comment]
relate: https://github.com/bentoml/BentoML/pull/954

**Pull request merged**